resources/i18n/en.json

```diff
@@ -156,6 +156,10 @@
     "move-files-to-trash-text": "This action will move the file to trash.",
     "perma-delete-files-text": "This action will permanently delete the file.",
     "clear-filter": "Clear the currently set filters",
-    "search-language": "Search language",
-    "search-category": "Search category"
+    "choose-language": "Filter language",
+    "choose-category": "Filter category",
+    "choose-content-type": "Filter content type",
```

Comment:
As this is used as placeholder, I would name them `language-searcher-placeholder`